core/Commands.js

```diff
@@ -63,7 +63,150 @@ var Commands = {
     DOM: 32,
     READY: 33,
     ALLOW_DEFAULT: 34,
-    PREVENT_DEFAULT: 35
+    PREVENT_DEFAULT: 35,
+    prettyPrint: function (buffer, start, count) {
+        var callback;
+        start = start ? start : 0;
+        var data = {
+            i: start,
+            result: ''
+        };
+        for (var len = count ? count + start : buffer.length ; data.i < len ; data.i++) {
+            callback = commandPrinters[buffer[data.i]];
+            if (!callback) throw new Error('PARSE ERROR: no command registered for: ' + buffer[data.i]);
+            callback(buffer, data);
+        }
+        return data.result;
+    }
+};
+
+var commandPrinters = [];
+
+commandPrinters[Commands.INIT_DOM] = function init_dom (buffer, data) {
```

Comment:
Can we have an object to look up the "arity" of the commands, e.g. something like this:

```js
var commandToArity = {
    DOM_RENDER_SIZE: 0,
    ADD_CLASS: 1,
    SUBSCRIBE: 1,
    CHANGE_CONTENT: 1,
    ...
};
```


or (in order to have the ability to pretty print parameter names):

```
{
    TIME: ['ms'],
    CHANGE_SIZE: ['width', 'height'],
    ...
}
```

I think this would be a more flexible approach than adding 30 functions.